mantidproject/mantid
# Refs #22594. trying to remove the empty subdirs

**Description of work.**
Some empty subdirs were added in  3209dcfff0cea9879f690380cad58248b77da1dc, which looks like git submodules but may not be initialized correctly. This causes conda build to fail. This PR simply removes those subdirs.

**To test:**

Build mantid as usual and make sure build does not fail

Fixes #22594 

Does this update require release notes?

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
